spk/deluge/src/core.conf

```diff
@@ -16,7 +16,7 @@
   "daemon_port": 58846, 
   "torrentfiles_location": "@download_dir@", 
   "max_active_limit": 8, 
-  "geoip_db_location": "/usr/local/deluge/share/GeoIP/GeoIP.dat", 
+  "geoip_db_location": "/usr/share/geoip/GeoIP.dat", 
```

Comment:
@hgy59 Notice I do not find that file on DSM 6.1 system, it is only available on DSM 6.2